salt/modules/napalm_network.py

```diff
@@ -1899,6 +1892,14 @@ def load_template(template_name,
     }
     loaded_config = None
     # prechecks
+    deprecated_args = ('template_user', 'template_attrs', 'template_group', 'template_mode')
+    for deprecated_arg in deprecated_args:
+        if template_vars.get(deprecated_arg):
+            salt.utils.versions.warn_until(
+                'Sodium',
+                ('The \'{arg}\' argument to \'net.load_template\' is deprecated '
+                 'and has been ignored').format(arg=deprecated_arg)
+            )
```

Comment:
Can we remove these args from template args?

Should be a simple `del template_vars[deprecated_arg]`

Reply:
Shall do.